Documentation/doc/scripts/compare_testsuites.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#Path to the CGAL_Documentation_build_directory/doc_output
+PATH_TO_DOC="$1"
+#path to the repository containing the output of this script for the reference documentation.
+DOC_REF="$2"
+#output in a new directory
```

Comment:
You should check the number of arguments provided to the script and that the directories exist. Also you should show some error messages in both cases.